rust/tools/authorization-logic/raksha_examples/VisionPipelineAuthLogic

```diff
@@ -75,36 +72,32 @@
         isAccessPath(pathX).
 }
 
-// Maybe we want this part of the policy to sit closer to the particles
-// because sometimes relations will refer to path names ?
-// (We can ostensibly do this as long as we can write
-// `claim <RelationName(...)>` in manifests)
-//
-// NOTE: maybe the "will" relations should refer to handles rather than tags, 
-// but the "may"s still refer to the tags, and we figure out which tags to
-// get permissions about based on the taint analysis (for now both still 
-// operate on the tags).
 "ImageDetector" says "ImageDetector"
-    will("do_pure_computation", "raw_video_tag").
+    will("do_pure_computation", "id_sensor_packet.camera_feed").
 "ImageSelector" says {
-    "ImageSelector" will("do_pure_computation", "raw_video_tag").
+    "ImageSelector" will("do_pure_computation", "is_detection_boxes").
     declassifies("is_select_image_id", "raw_video_tag").
     hasTag("is_select_image_id", "product_id_tag").
     objectSelected("EndUser", "is_selected_image_id").
 }
 "ProductIdOutput" says 
-    "ProductIdOutput" will("egress_to_shopping_service", "product_id_tag").
+    "ProductIdOutput" will("egress_to_shopping_service", "pio_remote_service").
```

Comment:
If this `will` annotation speaks of the `pio_remote_service` within the `VisionPipeline` recipe, shouldn't it have to indicate the recipe name as part of the `will` annotation? Perhaps this should be:

"ProductIdOutput" will("egress_to_shopping_service", "VisionPipeline.pio_remote_service").

or something similar.